packages/validation/src/loadbalancers.schema.ts

```diff
@@ -151,3 +153,85 @@ export const UpdateRouteSchema = object({
     otherwise: (o) => o.of(HTTPRuleSchema),
   }),
 });
+
+export const loadBalancerLabelValidation = object({
```

Comment:
Can we go ahead and build this creation schema? 

We don't need to finish it, but it would be nice to not have to come back and rename it

```suggestion
export const CreateLoadbalancerSchema = object({
```

Reply:
@bnussman-akamai  we already have on that uses loadBalancerLabelValidation `createLoadBalancerSchema` [(line 232)](https://github.com/linode/manager/pull/9848/files#diff-60da22ad8f261e4c8ecbe11e6989891a9b6dad115ed21c11d704209b908ef72aR232) Will reference this in Formik validation.